File: C_Programming/m008_looping/Looping01.c
//
// Created by Kelvin HT on 1/19/22.
//

#include <stdio.h>

int main() {

    // Arithmetic: 1, 2, 3, 4, 5, ...
    int n;
    printf("Input n : ");
    scanf("%d", &n);

    // Looping 'for'
    for (int x = 0; x < n; x++) {
        printf("%d ", (x + 1));
    }
    printf("\n");

}

File: DartProgramming/bin/m009_looping/Looping01.dart
//Looping 'for'

import 'dart:io';

void main() {
  //Arithmetic: 1, 2, 3, 4, ...
  print('Deret Arithmetic : ');
  stdout.write('Input n : ');
  int n = int.parse(stdin.readLineSync()!);

  for (int x = 1; x <= n; x++) {
    stdout.write('$x, ');
  }
}